src/components/product-menu-dropdown/product-menu-dropdown.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from '@mapbox/react-icon';
+
+class ProductMenuDropdown extends React.PureComponent {
+  render() {
+    const categoryLength = this.props.categories.length;
+    const activeDotStyle = {
+      content: '',
+      width: '6px',
+      height: '6px',
+      borderRadius: '3px',
+      background: '#4264fb',
+      position: 'absolute',
+      margin: '10px 0 0 -12px',
+      display: 'none'
+    };
+    const allCategories = this.props.categories.map((category, index) => {
+      const allProducts = category.products.map((product, index) => {
+        const liClasses = classnames('mt6 relative ', {
+          'txt-bold': product.url && location.pathname.indexOf(product.url) > -1
+        });
+        const dotClasses = classnames({
+          'inline-block': location.pathname.indexOf(product.url) > -1,
+          none: location.pathname.indexOf(product.url) < 0
+        });
+        return (
+          <li className={liClasses} key={index}>
+            <div style={activeDotStyle} className={dotClasses} />
+            <a href={product.url} className="color-blue-on-hover">
+              {product.name}
+            </a>
+          </li>
+        );
+      });
+      const categoryClasses = classnames('wmin240 py24', {
+        'border-b border--gray-light': index < categoryLength - 1
+      });
+      return (
+        <div className={categoryClasses} key={index}>
+          <div className="txt-bold color-gray">
+            <Icon
+              name={category.icon}
+              inline={true}
+              className="icon mb-neg6 mr6 h20 w20 color-gray inline"
```

Comment:
Is `inline` necessary? Or is it covered by `inline={true}`?

Reply:
@davidtheclark looks like they're both needed: 

- `.inline` sets `display: inline !important`
- `inline={true}` does some math to position the icon vertically: https://github.com/mapbox/mapbox-react-components/blob/master/packages/react-icon/src/icon.js#L19-L28